topydo/commands/ListCommand.py

```diff
@@ -69,11 +70,14 @@ def _process_flags(self):
                     self.printer = None
             elif opt == '-F':
                 self.format = value
+            elif opt == '-N':
+                # 2 lines are assumed to be taken up by printing the next prompt
+                self.limit = get_terminal_size().lines - 2
```

Comment:
What if `get_terminal_size().lines() < 2`? I think the value should be `max()`'ed to 0.
I'd like to see this covered in a test case.